src/components/navbutton.js

```diff
@@ -0,0 +1,91 @@
+/** @jsx jsx */
+import { Box, Flex, NavLink, Image, jsx } from 'theme-ui'
+import { Component } from 'react'
+import React from 'react'
+import { BsCaretDownFill, BsCaretUpFill } from 'react-icons/bs'
+
+class NavButton extends Component {
+  constructor(props) {
+    super(props);
+    const {loc,text,icon,active,dropdown} = props;
+    this.loc = loc;
+    this.text = text;
+    this.active = active;
+    this.icon = icon;
+    this.dropdown = dropdown;
```

Comment:
Why are props copied (and never updated)? Please make this a function component.